docs/markdown/Qt5-module.md

```diff
@@ -51,6 +51,29 @@ This method takes the following keyword arguments:
 
 *Since: 0.54.0*
 
+## is_static
+This method returnes `true` if the Qt install is statically built.
+
+*Since 0.56.0*
+
+## static_plugins_dep
+When linking against a statically built Qt installation, plugins also needs static linking. The `static_plugins_dep` return a dependecy object that holds the nessecary
+linker flags for static linking, and generates files that includes the plugins using Qt's `Q_IMPORT_PLUGIN`. Normal plugins (those found in the `plugins` folder) must
+needs to be specified manually, while QML plugins are automatically detected from QML and JavaScript files.
+
+The method takes the following keyword arguments:
+- `qresources`, rcc source file used for scanning for QML plugins
```

Comment:
A `qmlfiles` argument is needed for applications that distribute loose QML files instead of bundling them into the binary with the Qt resource compiler.

Reply:
I can certainly add that, although bundling loose QML files would kind of defeat the purpose of linking statically.